Presentation.Web/app/components/local-config/import/fk-organization-import-change-logs-root.component.ts

```diff
@@ -0,0 +1,77 @@
+﻿module Kitos.LocalAdmin.Components {
+    "use strict";
+
+    function setupComponent(): ng.IComponentOptions {
+        return {
+            bindings: {
+                organizationUuid: "@"
+            },
+            controller: FkOrganizationImportChangeLogRootController,
+            controllerAs: "ctrl",
+            templateUrl: `app/components/local-config/import/fk-organization-import-change-logs-root.view.html`
+        };
+    }
+
+    interface IFkOrganizationImportChangeLogRootController {
+        organizationUuid: string;
+    }
+
+    class FkOrganizationImportChangeLogRootController implements IFkOrganizationImportChangeLogRootController {
+        organizationUuid: string | null = null;
+
+        changeLogs: Array<Models.Api.Organization.ConnectionChangeLogDTO> = [];
+        selectedChangeLog: Models.Api.Organization.ConnectionChangeLogDTO;
```

Comment:
remember default values for these

Reply:
hmm... are you sure about the type? As far as I can see select2 works on select2 vms...
it's cheating with the correlated id here..